src/engine/game/world/events/npc.lua

```diff
@@ -36,6 +36,8 @@ function NPC:init(actor, x, y, properties)
     self.set_value = properties["setvalue"]
 
     self.interact_count = 0
+
+    self.buffer = 5
```

Comment:
```suggestion
    self.interact_buffer = (5/30)
```
Was not adjusted to be in seconds